internal/backends/qt/qt_widgets/button.rs

```diff
@@ -197,7 +197,7 @@ impl Item for NativeButton {
     fn layout_info(
         self: Pin<&Self>,
         orientation: Orientation,
-        _window: &WindowInner,
+        _platform_window: &PlatformWindowRc,
```

Comment:
in general, i think it's ok to keep the name window  instead of platform_window for the variable name. Later i wonder how we will rename this trait, but i think PlatformWindow is not the right name. (We mentioned WindowBackend)

Reply:
I thought about it and wend back and forth a bit during the hacking on this series and I've noticed two things:

1. It really helps for code readability to not use the term "window" when the type is the platform window. Using the same made code definitely harder to read, so I for one don't want to change it - unless we rename "PlatformWindow" to something else of course.
2. I've come to quite like the name `PlatformWindow` because it really resembles the platform specific aspects of a Window. `WindowBackend` emphasises the backend (which is in turn a very generic term), while here I think that we can afford to be specific - this is about platform specific adaptations.

However I also see that we have introduced the term "backend" in Slint early on and it's visible in documentation and cargo/cmake features. I'm in favor of having a separate discussion about those names before going public. I'll file #1503.